### ISSUE TITLE
[BUG] Kirby-button in kirbyHeaderActions not showing label on mobile size screens

### ISSUE THREAD

**Issue by hjorte-bd** [kirbydesign/designsystem#3299]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

When a kirby button is placed in `kirbyHeaderActions` in a header, it is not possible for label to be shown, even when there is nothing else in header. Our designer would like this to be possible.

## Describe the bug
```
  <kirby-header [title]="'menu.title' | translate">
    <kirby-action-group *kirbyHeaderActions>
      <button
        kirby-button
        attentionLevel="3"
        [class.icon-only]="false"
        (click)="logout()"
      >
        <kirby-icon name="log-out"></kirby-icon>

        <span class="text">{{ 'menu.logout' | translate }}</span>
      </button>
    </kirby-action-group>
  </kirby-header>
```
## Describe how to reproduce the bug

1. Create `kirby-header` element
2. Inside, add `kirby-action-group` element wiith a `*kirbyHeaderActions` directive
3. Inside add kirby button
4. Label text will not be shown on small screens
4a. `[class.icon-only]="false"` will show the text, but will result in incorrect layout.

## Which Kirby version was used?
8.11.2 (Also v9)

## What was the expected behavior?
The text should be shown when there is plenty of space, or at least be possible to force to be shown anyways (like with ` [class.icon-only]="false"`)

## Add any screenshots
How it looks when text is forced back by removing text-only class.
![image](https://github.com/kirbydesign/designsystem/assets/110398431/911a3d16-cfaf-4560-96b8-0984f11f156a)

## Please complete the following information:
- OS: MacOS
- Browser: Edge
- Browser version: 119

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/main/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
This is intended behavior for the action-group, you should use custom actions instead as described in the header documentation.

**Comment by jakobe** [comment]
Re-opening this, as it's not possible to opt out of the collapsed (icon only) button even with custom actions.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
